examples/sha256/main.rs

```diff
@@ -169,6 +168,5 @@ impl<F: FieldExt, Sha256Chip: Sha256Instructions<F>> Sha256<F, Sha256Chip> {
         hasher.finalize(layouter.namespace(|| "finalize"))
     }
 }
-*/
 
 fn main() {}
```

Comment:
This example still doesn't do anything.

Reply:
Yes, the actual test code is in `examples/sha256/benches.rs`. But I think we chose to exclude it from `sha256/main.rs` because it would slow down CI significantly.